link/src/main/java/com/meshconnect/link/entity/AccessTokenPayload.kt

```diff
@@ -15,7 +15,7 @@ data class AccessTokenPayload(
     val brokerBrandInfo: BrandInfo,
     val expiresInSeconds: Int? = null,
     val refreshTokenExpiresInSeconds: Int? = null,
-) : FrontPayload
+) : LinkPayload
 
 @Parcelize
 data class AccountToken(
```

Comment:
As we decided to stick with Web SDK, I suggest to keep payload structure names and fields the same across platforms
BrokerAccountToken
BrokerAccount
BrokerBrandInfo
LinkPayload (need to rename FrontPayload in Web SDK)
AccessTokenPayload
DelayedAuthPayload

Reply:
There is a doc where we keep the names. I will dm you.

Reply:
@AlexZeml-Front added missing fields